### PR TITLE
Correct doc text indicating availability of repository rather than of release

### PR DESCRIPTION
**Pull request by robbrockbank**

### CODE REVIEW

File: docs/ManualClusterSetup.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Get the calico binary onto each host. It's usually safe to just grab the latest
     wget http://projectcalico.org/latest/calicoctl
 	chmod +x calicoctl
 
-The beta is not HA so may not be suitable for production environments.  Alternatively, you can download a specific [release](https://github.com/Metaswitch/calico-docker/releases/).  For example, to retrieve the most recent release, on each host run
+Note that projectcalico.org is not a HA repository, so using this download URL is not recommended for any automated production installation process.  Alternatively, you can download a specific [release](https://github.com/Metaswitch/calico-docker/releases/) from github.  For example, to retrieve the v0.4.2 release, on each host run
```
Comment (Symmetric) [inline docs/ManualClusterSetup.md]: => 'an HA'

File: docs/Orchestrators.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Get the calico binary onto each node. It's usually safe to just grab the latest
     wget http://projectcalico.org/latest/calicoctl
 	chmod +x calicoctl
 
-The latest beta is not HA so may not be suitable for production environments.  Alternatively, you can download a specific [release](https://github.com/Metaswitch/calico-docker/releases/), e.g.
+Note that projectcalico.org is not a HA repository, so using this download URL is not recommended for any automated production installation process.  Alternatively, you can download a specific [release](https://github.com/Metaswitch/calico-docker/releases/) from github.  e.g.
```
Comment (Symmetric) [inline docs/Orchestrators.md]: => "an HA"

**Comment by Symmetric** [comment]
Minor typo, but LGTM apart from that

**Comment by robbrockbank** [comment]
Fixes #133

**Pull request merged**